lark/lark.py

```diff
@@ -401,6 +410,28 @@ def open(cls, grammar_filename, rel_to=None, **options):
             grammar_filename = os.path.join(basepath, grammar_filename)
         with open(grammar_filename, encoding='utf8') as f:
             return cls(f, **options)
+    
+    @classmethod
+    def open_from_package(cls, package, grammar_path, search_paths=("",), **options):
+        """Create an instance of Lark with the grammar loaded from within the package `package`.
+        This allows grammar loading from zipapps.
+        
+        Will also create a `FromPackageLoader` instance and add it to the `import_sources` to simplify importing
```

Comment:
Why is this useful for the user to know?

Reply:
I thought that it would be useful to prevent users from manually specific the `import_sources`. `FromPackageLoader` is after all a public class.

Reply:
Hmm maybe it's better to say something like 

    Imports in the grammar will use the `package` and `search_paths` provided, through `FromPackageLoader`.

But I'll leave it to your judgement.